treeherder/seta/preseed.py

```diff
@@ -58,9 +56,6 @@ def load_preseed(validate=False):
     the * field. For example: (linux64, pgo, *) matches all Linux 64 pgo tests
     """
     logger.info("About to load preseed.json")
-    if not JobPriority.objects.exists():
```

Comment:
Removing this permits Heroku Review Apps to initialize properly since they were trying to run `load_preseed` before `initialize_seta`.

I also don't see a reason why we would want to prevent loading preseed data before running initialize_seta (which loads data from the runnable jobs artifact).

Reply:
this is the only area I am not sure if we need it or not; since we are keeping a close eye on things right now, it seems ok to go forward with this.